tests/golden/set-field-command/set-field-command.bats

```diff
@@ -14,7 +14,10 @@ load '../helpers'
   run coffer set-field /a/b test aba -V public
```

Comment:
Please add a test where we use `set-field` to create a multi-line field, and the output of `set-field` contains the fix you added to this PR.

Reply:
Done